pkg/sourcereader/sourcereader_test.go

```diff
@@ -51,7 +51,7 @@ func setup() {
 	dirName := fmt.Sprintf("ghpc_sourcereader_test_%s", t.Format(time.RFC3339))
 	dir, err := os.MkdirTemp("", dirName)
 	if err != nil {
-		log.Fatalf("sourcereader_test: %v", err)
+		logging.Fatal("sourcereader_test: %v", err)
```

Comment:
Giving it a second thought, there is no reason to use custom logging in `*_test.go` files.
If you agree, please reset all test files.

Reply:
To clarify, I don't think that `log.Fatalf` is an appropriate way to terminate tests neither, it should be done by means of test library, but this is out of scope of this PR.

Reply:
I will switch them. If we ever want custom logging for the test files, we can always include them back in.

Reply:
#1985  for you to review

Reply:
I switched them back so custom logging isn't in any `*_test.go` files.